prow/flagutil/kubernetes_cluster_clients.go

```diff
@@ -206,6 +208,42 @@ func (o *KubernetesOptions) BuildClusterCoreV1Clients(dryRun bool) (v1Clients ma
 	return clients, nil
 }
 
+// BuildClusterManagers returns a manager per buildCluster.
+// Per default, LeaderElection and Metrics are disabled, as we assume
```

Comment:
What does `Metrics` give us? Would be nice to see e.g. queue depth per build cluster. Unless `Metrics` just needs to be on *one* manager in the process

Reply:
Yeah, this is just about a metric serving endpoint, not about the metrics themselves. C-R has its own metrics package with a registry, but we make our metrics serve both the ones from the prometheus registry and from the c-r registry.

I'll clarify the comment.